external/src/main/java/com/redhat/cloud/policies/engine/metrics/ProcSelfStatusExporter.java

```diff
@@ -33,15 +36,31 @@ public class ProcSelfStatusExporter {
 
     private boolean hasWarned = false;
 
-    private long vmHwm;
-    private long vmRss;
-    private long rssAnon;
-    private long rssFile;
-    private long vmStk;
-    private long vmLib;
-    private long vmData;
-    private long vmSize;
-    private int threads;
+    private final AtomicLong vmHwm = new AtomicLong(0);
+    private final AtomicLong vmRss = new AtomicLong(0);
+    private final AtomicLong rssAnon = new AtomicLong(0);
+    private final AtomicLong rssFile = new AtomicLong(0);
+    private final AtomicLong vmStk = new AtomicLong(0);
+    private final AtomicLong vmLib = new AtomicLong(0);
+    private final AtomicLong vmData = new AtomicLong(0);
+    private final AtomicLong vmSize = new AtomicLong(0);
+    private final AtomicInteger threads = new AtomicInteger(0);
+
+    @Inject
+    MeterRegistry meterRegistry;
+
+    @PostConstruct
+    public void initCounters() {
+        meterRegistry.gauge("status.vmHwm", vmHwm);
+        meterRegistry.gauge("status.vmRss", vmRss);
+        meterRegistry.gauge("status.rssAnon", rssAnon);
+        meterRegistry.gauge("status.rssFile", rssFile);
+        meterRegistry.gauge("status.vmStk", vmStk);
+        meterRegistry.gauge("status.vmLib", vmLib);
+        meterRegistry.gauge("status.vmData", vmData);
+        meterRegistry.gauge("status.vmSize", vmSize);
+        meterRegistry.gauge("status.threads", threads);
```

Comment:
The old tags are missing. Units too but I don't know if Micrometer supports that.